documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -4,16 +4,10 @@
 
 ### New features
 
-* 
-
-### Changed features
-
-* 
-
-### Resolved issues
-
-* 
+* [company_name] [solution_name] now includes the maven embedded or shaded dependencies as part of the Bill of Materials (BOM) via the property --detect.maven.include.shaded.dependencies. See the [detect.maven.include.shaded.dependencies](properties/detectors/maven.md#maven-include-shaded-dependencies) property for more information.
```

Comment:
Is this true for all Maven detectors (Maven CLI, Maven Wrapper CLI and Project Inspector). If not, then include the applicable detector names.

Reply:
Yes, this is true for all the Maven Detectors.